frameworks/react/__tests__/AdvancedVideo.test.tsx

```diff
@@ -116,4 +116,14 @@ describe('AdvancedVideo', () => {
       done();
     }, 0);// one tick
   });
+
+  it('Should support forwarding ref to underlying video element', () => {
+    const myRef = React.createRef();
+    mount(<AdvancedVideo cldVid={cloudinaryVideo} ref={myRef} />);
```

Comment:
Shouldn't we allow the user to also access the component itself with a ref?

Reply:
don't think so, this component renders a video element with no wrapper

Reply:
Isn't this decision both final and painful to change in the future?

If we decide that we do want to allow users to get a ref to the component, this will be a messy change as `ref` is already used to get the videoElementRef, so we'll need to create a new prop for the non-standard behavior.


Would it be clearer to the user if he had two props to pass? `ref` and `videoElementRef`?
Where `ref` behaves as expected in React, and `videoElementRef` gets the videoElement.